.travis.yml

```
@@ -1,10 +1,7 @@
 language: python
 
-sudo: yes
-dist: xenial
-
 python:
-  - 3.7
+  - "3.7-dev"
 
 install:
   - pip install tox-travis codecov
```

```suggestion
  - pip install codecov
```

A dependência pode ser removida também, ela estava sendo instalada somente para rodar o tox.